consensus/chronology/chronology_test.go

```diff
@@ -325,7 +325,7 @@ func TestChronology_SetAppStatusHandlerWithOkValueShouldPass(t *testing.T) {
 func TestChronology_CheckIfStatusHandlerWorks(t *testing.T) {
 	t.Parallel()
 
-	chanDone := make(chan bool, 1)
+	chanDone := make(chan bool, 2)
```

Comment:
what does 2 means ?

Reply:
We need a buffered channel because channel is set 2 times in test.